### DISCUSSION TITLE
반성합니다 반성합니다

### DISCUSSION THREAD

**Discussion by gusah009** [BDD-CLUB/challenge-every-week-posting#546] [6. 반성문]
롤에 빠져버린 악마같은 삶... 누가 좀 꺼내주세요

<img width="649" alt="image" src="https://github.com/BDD-CLUB/challenge-every-week-posting/assets/26597702/6b6c66f3-67e6-4a18-8944-4b1476a5faf7">

**Comment by shkisme** [comment]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ 현충일날 달리셨군요

**Comment by stopmin** [comment]
저거 혹시 판 수인가요????
